reloj-checador/app/helpers/checkins_helper.rb

```diff
@@ -0,0 +1,16 @@
+module CheckinsHelper
+  def flash_msg
```

Comment:
no need to duplicate this helper, all helpers located in application_helper are available everywhere

Reply:
Commend taken and helper deleted.